src/AvailabilityManagerInterface.php

```diff
@@ -40,8 +45,8 @@ public function getCheckers();
    * @param \Drupal\commerce\Context $context
    *   The context.
    *
-   * @return bool
-   *   TRUE if the purchasable entity is available, FALSE otherwise.
+   * @return \Drupal\commerce\AvailabilityResponseInterface
```

Comment:
I ended up moving these AvailabilityResponse* classes into the root Drupal\commerce namespace.